mods/seasonal/moves.js

```diff
@@ -475,6 +474,29 @@ exports.BattleMovedex = {
 		target: "allAdjacent",
 		type: "Electric",
 	},
+	// Teremiare
+	batonthief: {
+		accuracy: true,
+		category: "Status",
+		shortDesc: "Steals target's boosts and then Baton Passes them out.",
+		id: "batonthief",
+		isNonstandard: true,
+		name: "Baton Thief",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		stealsBoosts: true,
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
+			this.add('-anim', source, "Spectral Thief", target);
+		},
+		onAfterMoveSecondarySelf: function (source) {
+			this.useMove('batonpass', source);
```

Comment:
We shouldnt call other moves unless needed.

Reply:
Joim noted this in the code for ssbm:
> // selfSwitch: 'copyvolatile' does not work with targeted moves.
So calling the move seems to be the only way.

Reply:
ok